cloudify_helm/constants.py

```diff
@@ -20,14 +20,18 @@
 HELM_CONFIG = "helm_config"
 API_OPTIONS = "api_options"
 VALUES_FILE = "values_file"
+AWS_CLI_VENV = "aws_cli_venv"
 CONFIGURATION = "configuration"
 CLIENT_CONFIG = "client_config"
-AUTHENTICATION = 'authentication'
+AUTHENTICATION = "authentication"
 RESOURCE_CONFIG = "resource_config"
 EXECUTABLE_PATH = "executable_path"
 DATA_DIR_ENV_VAR = "HELM_DATA_HOME"
 CACHE_DIR_ENV_VAR = "HELM_CACHE_HOME"
+AWS_CLI_TO_INSTALL = "awscli==1.19.35"
```

Comment:
What's significant about this version?

Reply:
They are going to deprecate for python2 so just to be on the safe side.
https://docs.aws.amazon.com/cli/latest/userguide/install-linux.html